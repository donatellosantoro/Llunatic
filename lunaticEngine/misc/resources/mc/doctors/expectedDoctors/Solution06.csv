#doctor
npi,name,hospital,email
_L29,G. House,_L168,house@ppth.com
_L472,_L478,Sacred Hearth,chase@sh.com

#prescription
rxid,patient,npiref
P1,J. Doe,_L472
P2,W. Smith,_L29
P3,M. White,_L29

