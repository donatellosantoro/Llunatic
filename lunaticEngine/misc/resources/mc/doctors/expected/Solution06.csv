#prescription
rxid,patient,npiref
P3,M. White,_L15
P1,J. Doe,_L343
P2,W. Smith,_L15

#doctor
npi,name,hospital,email
_L343,_L347,Sacred Hearth,chase@sh.com
_L15,G. House,PPTH,_L323

